infrastructure/main.tf

```diff
@@ -1,3 +1,41 @@
 provider "azurerm" {
   version = "=1.33.1"
 }
+
+locals {
+  reform-scan-vault-name = "reform-scan-${var.env}"
+  bulk-scan-vault-name = "bulk-scan-${var.env}"
+}
+
+data "azurerm_key_vault" "bulk-scan_key_vault" {
+  name                = "${local.bulk-scan-vault-name}"
+  resource_group_name = "bulk-scan-${var.env}"
+}
+
+data "azurerm_key_vault_secret" "cft_storage_account_name" {
```

Comment:
perhaps we don't need to know about the abbreviations here? shall we just prefix with `bulk_scan` instead?